test/dlc_tests/conftest.py

```diff
@@ -952,7 +976,7 @@ def generate_unique_values_for_fixtures(
         "autogluon": "ag",
     }
     fixtures_parametrized = {}
-
+    print(f"imgs: {images_to_parametrize}, {metafunc_obj.fixturenames}")
```

Comment:
Is this meant to be a permanent addition to the code, or is it only here temporarily for the sake of debugging?

I don't see any output logs in the test jobs that contain `imgs: `